relational_operators/TextScanOperator.cpp

```diff
@@ -19,6 +19,12 @@
 
 #include "relational_operators/TextScanOperator.hpp"
 
+#include "relational_operators/RelationalOperatorsConfig.h"  // For QUICKSTEP_HAVE_UNISTD.
```

Comment:
@hbdeshmukh No, this line is ok, as the next line immediately uses `QUICKSTEP_HAVE_UNISTD` defined in the header file.

See https://github.com/apache/incubator-quickstep/blob/master/storage/StorageManager.cpp#L20 for more info and examples.